packages/block-editor/src/components/block-tools/block-selection-button.js

```diff
@@ -252,8 +285,13 @@ function BlockSelectionButton( { clientId, rootClientId } ) {
 					<BlockIcon icon={ icon } showColors />
 				</FlexItem>
 				<FlexItem>
-					{ editorMode === 'zoom-out' && (
-						<BlockMover clientIds={ [ clientId ] } hideDragHandle />
+					{ editorMode === 'zoom-out' && ! isBlockTemplatePart && (
+						<BlockMover
+							clientIds={ [ clientId ] }
+							hideDragHandle
+							isPrevBlockTemplatePart={ isPrevBlockTemplatePart }
+							isNextBlockTemplatePart={ isNextBlockTemplatePart }
```

Comment:
I wonder if these names are too specific. Should we just be passing `blockMoverUpButtonDisabled` and `blockMoverDownButtonDisabled` and letting the logic live in this file instead? There might be other instances where these buttons are enabled/disabled so a more generic prop might be useful.

Reply:
Another correct observation, will apply